tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,229 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Icon,
+  Flex,
+  Button,
+  Section,
+  Collapsible,
+} from "../components";
+import { Window } from "../layouts";
+import { logger } from "../logging";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+  
+  return (
+    <Window
+      resizable
+      title={`Vote${
+        mode
+          ? `: ${
+            question
+              ? question.replace(/^\w/, c => c.toUpperCase())
+              : mode.replace(/^\w/, c => c.toUpperCase())
+          }`
+          : ""
+      }`}
+      width={400}
+      height={500} >
+      <Window.Content overflowY="scroll">
+        <Flex direction="column" height="100%">
+          {!!lower_admin && <AdminPanel />}
+          <ChoicesPanel />
+          <TimePanel />
+        </Flex>
+      </Window.Content>
+    </Window>
+  );
+};
+
+// Collapsible panel for admin actions.
+const AdminPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { avm, avr, avmap, voting, upper_admin } = data;
+  return (
+    <Flex.Item>
+      <Section mb={1} title="Admin Options">
+        <Collapsible title="Start a Vote">
+          <Flex mt={2} justify="space-between">
+            <Flex.Item>
+              <Box mb={1}>
```

Comment:
Here's a couple ideas to freshen up this collapsible, if you want.

Moving checkboxes before vote buttons will allow all the buttons to line up cleanly
![image](https://user-images.githubusercontent.com/40974010/111526961-9430eb00-871c-11eb-8808-49ee5f700c52.png)

putting all the things in this collapsible in a flex (or stack, you should consider converting all this to a stack brother 🤠 ) will let you set a basis on the first item (being the vote options) which lets you space out the disabled checkboxes evenly
![image](https://user-images.githubusercontent.com/40974010/111527896-acedd080-871d-11eb-897b-eef6981173a9.png)

Reply:
Also is it just me, or these checkboxes should be **un**inverted? `[✔️] Enabled`, not `[✔️] Disabled`.

Reply:
Brain being a fuck here, I think I tried keeping it true to source because I wasn't sure of the use case for disabling these. Sounds a bit more logical that way so I'll fix it